lib/modules/Server.js

```diff
@@ -13,6 +13,31 @@ import actions from './actions';
 import { createQuery } from './query';
 import { METHODS } from './router';
 
+export const dispatchInitialShell = async (ctx, dispatch) => {
```

Comment:
when i mean separate module, i mean another modules within `@r/platform`, not yet another npm module :)